configcatclient/configfetcher.py

```diff
@@ -49,10 +49,12 @@ def is_not_modified(self):
 
 class ConfigFetcher(object):
     def __init__(self, sdk_key, mode, base_url=None, proxies=None, proxy_auth=None,
-                 data_governance=DataGovernance.Global):
+                 connect_timeout=10, read_timeout=30, data_governance=DataGovernance.Global):
```

Comment:
I think we can trait `connect_timeout` and `read_timeout` parameters required here without default values as those are already defined at the initialization of the client. We may should keep them at a single place to maintain.

Reply:
Discussed. We will keep the default values here.